contracts/protocol/libs/ProtocolLib.sol

```diff
@@ -182,6 +182,8 @@ library ProtocolLib {
         mapping(uint256 => BosonTypes.AuthToken) pendingAuthTokenUpdatesBySeller;
         // dispute resolver id => DisputeResolver
         mapping(uint256 => BosonTypes.DisputeResolver) pendingAddressUpdatesByDisputeResolver;
+        // groupId => tokenId => commit count (count how many times a token has been used as gate for this group)
+        mapping(uint256 => mapping(uint256 => uint256)) conditionalCommitsByTokenId;
```

Comment:
Not that it matters that much, but `conditionalCommitsByAddress` has just different mapping order
`address => groupId => commit count`, (so groupId comes after the address), while in `conditionalCommitsByTokenId` it comes before `tokenId`.

Performance-wise it think there is no difference, and probably your choice is a bit more natural, but I think that we should be consistent here and apply the same order in both cases. And since by address lookup is already there, the new one should be changed.